Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.4-alpine AS build_deps
+FROM golang:1.15-alpine AS build_deps
```

Comment:
1.16

Reply:
👍 In that case, GO111MODULE can go away too